raiden/tests/unit/test_cli.py

```diff
@@ -68,7 +68,9 @@ def run_test_check_json_rpc_geth():
 
 def test_check_json_rpc_geth():
     # Pin the highest supported version for the test purposes
-    with patch("raiden.utils.ethereum_clients.HIGHEST_SUPPORTED_GETH_VERSION", new="1.9.2"):
+    with patch("raiden.utils.ethereum_clients.HIGHEST_SUPPORTED_GETH_VERSION", new="1.9.2"), patch(
+        "raiden.utils.ethereum_clients.LOWEST_SUPPORTED_GETH_VERSION", new="1.7.2"
```

Comment:
This should be `1.8.21`

Reply:
no. We can't change the unit tests every time we change the supported versions. These tests only test the version check function

Reply:
I don't understand. The min supported version is 1.8.21, but here in the test you're patching it to 1.7.2 but i can't see that a failure is expected.

Reply:
So the test has various value inside it. It was initially written with the global values in mind. But as we slowly change the low/high supported versions we should not have to edit the test.

Instead let's have them fixed as patches.

The expected failures/sucesses can be seen in the test itself: https://github.com/raiden-network/raiden/blob/abc8c0006c77b3434cd26029be628b3050eb2546/raiden/tests/unit/test_cli.py#L37-L66

Reply:
Ahh i see. If you patch the value to the new minimum, every value in `run_test_check_json_rpc_geth` will have to changed. Ok discard my comment.